src/ignitus-Profile/ignitus-StudentProfile/Components/profileDetailView.tsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import { StyledHeading2, StyledHeading4, StyledHeading6 } from '../../../ignitus-UserInterfaceBook/styles';
+import * as C from '../styles';
```

Comment:
We use `C` generally for importing colors, can you please user `P` for Profile

Reply:
sure!